packages/launcher/lib/windows/index.ts

```diff
@@ -203,15 +207,21 @@ export function getPathData (pathStr: string): PathData {
     const pathParts = path.split(':')
 
     browserKey = pathParts.pop() || ''
-    path = pathParts.join(':')
+    path = doubleEscape(pathParts.join(':'))
 
     return { path, browserKey }
   }
 
+  path = doubleEscape(path)
+
   if (pathStr.indexOf('chrome.exe') > -1) {
     return { path, browserKey: 'chrome' }
   }
 
+  if (pathStr.indexOf('edge.exe') > -1) {
+    return { path, browserKey: 'edge' }
```

Comment:
Ha, I guess this does technically work for msedge.exe too

Reply:
Yeah, keeping that `browserKey` edge for msedge.exe too as we use `edge` everywhere else.